### PR TITLE
Add GceInstanceInfo structure to Attestation

### PR DESCRIPTION
**Pull request by josephlr**
Also add comments explaining what the attestation fields do.

Also add an `InstanceURL()` method to make it easier to call [`getShieldedInstanceIdentity`](https://cloud.google.com/compute/docs/reference/rest/v1/instances/getShieldedInstanceIdentity)

### CODE REVIEW

File: proto/tpm.proto [hunk 1]
```diff
@@ -47,8 +50,24 @@ message Pcrs {
   map<uint32, bytes> pcrs = 2;
 }
 
+// Information uniquely identifying a GCE instnace. Can be used to create an
```
Comment (iKevinY) [inline proto/tpm.proto:53]: s/instnace/instance/ (also in PR description).

**Review by iKevinY — APPROVED** [review]

File: proto/tpm.proto (hunk 1 above)
Comment (josephlr) [reply proto/tpm.proto:53]: Bahhhhh, thank you

Comment (iKevinY) [reply proto/tpm.proto:53]: Oh also s/Attestaion/Attestation in the PR title (in case that gets used as the squashed commit message lol).

Comment (josephlr) [reply proto/tpm.proto]: Lol, should be fixed now

**Pull request merged**